requirements/requirements.txt

```diff
@@ -6,6 +6,7 @@ python-dateutil
 ruamel.yaml
 scikit-learn
 setuptools==59.5.0
+tensorboard
```

Comment:
Why are we adding this?

Reply:
We don't want tensorboard at all cause it's causing ASR to segfault due to Protobuf incompatibility. Either remove it or use the latest version so that it fixes Protobuf.